Tests/RxSwift/MapAtTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MapToKeyPath.swift
+//  RxSwiftExt-iOS
+//
+//  Created by Michael Avila on 2/8/18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+
+import RxSwift
+import RxSwiftExt
+import RxTest
+
+struct Person {
```

Comment:
You can move this struct inside the MapAtTests class - It's not needed outside of the scope of this file/test